containers/Balancer.ts

```diff
@@ -33,10 +33,23 @@ interface PoolTokenQuery {
   balance: string;
 }
 
-const YIELD_TOKENS = [
-  "0x81ab848898b5ffD3354dbbEfb333D5D183eEDcB5", // Sep20
-  "0xB2FdD60AD80ca7bA89B9BAb3b5336c2601C020b4", // Oct20
-];
+interface YIELD_TOKEN {
+  [key: string]: string[];
+}
+const YIELD_TOKENS: YIELD_TOKEN = {
+  "0x81ab848898b5ffD3354dbbEfb333D5D183eEDcB5": [
```

Comment:
nit: why to lower case instead of to checksum address?

Reply:
the input to the graphQL query only accepts lower case addresses

Reply:
I cleaned this up a bit by pulling toLowerCase to the set function.